px-data-grid.html

```diff
@@ -1885,17 +1908,58 @@
               selected.push(key);
             }
 
-            if (!this.hideColumnFilter) {
-              content.push(item);
-            }
+            content.push(item);
           });
 
-          this._actionMenuContent = content;
+          this._columnSelectorContent = content;
           setTimeout(() => {
             this._selectedActionItems = selected;
           });
         }
 
+        /**
+         * Returns a label for the column selector, updated when column selections change.
+         */
+        _computeColumnLabel(selectedItems) {
```

Comment:
Not a big deal, but the function name is a little misleading, I figure `_computeColumnSelectorLabel` would be better -- albeit a bit long...

Reply:
Updated.